### ISSUE TITLE
mug on top left corner instead of the steam 

### ISSUE THREAD

**Issue by guifranzmann** [The-Ludwig/Scientifika#15]

**Comment by guifranzmann** [comment]
Now I'm thinking about removing the logo from the top left corner, as it actually appears below anwyays.

**Comment by The-Ludwig** [comment]
@guifranzmann Yes, I also thought about that, since the mug itself is actually not really scientifika-specific, without the steam...
